src/static/js/fb-ctf.js

```diff
@@ -2464,6 +2501,41 @@ function setupInputListeners() {
       event.preventDefault();
       $(this).onlySiblingWithClass('active');
     });
+
+    // custom logo file selector
+    var $customEmblemInput = $('#custom-emblem-input');
+    var $customEmblemPreview = $('#custom-emblem-preview');
+    var $customEmblemCarouselNotice = $('#custom-emblem-carousel-notice');
+    $('#custom-emblem-link').on('click', function() {
+      $customEmblemInput.trigger('click');
+    });
+    // on file input change, set image preview and emblem carousel notice
+    $customEmblemInput.change(function() {
+console.log('foo');
+      var input = this;
+      if (input.files && input.files[0]) {
+        if (input.files[0].size > (500*1024)) {
```

Comment:
I think this should be 1000 bytes. We're using 1000 in the server side code, at least.